rpc/nodeagent/network/virtualnetwork.pb.go

```diff
@@ -7,12 +7,12 @@ import (
 	context "context"
 	fmt "fmt"
 	proto "github.com/golang/protobuf/proto"
+	empty "github.com/golang/protobuf/ptypes/empty"
+	wrappers "github.com/golang/protobuf/ptypes/wrappers"
 	common "github.com/microsoft/moc/rpc/common"
 	grpc "google.golang.org/grpc"
 	codes "google.golang.org/grpc/codes"
 	status "google.golang.org/grpc/status"
-	emptypb "google.golang.org/protobuf/types/known/emptypb"
```

Comment:
FYI: It seems like a previous change accidentally used the wrong Go runtime libraries version when generating these files. This change ensures this file is consistent with the other `*.pb.go` files.